release/src/main.py

```diff
@@ -282,13 +282,21 @@ def upload(release_request,artifactory_repo,gid,aid,qual,ext,version):
   else:
     print("no attachment to github release")
   #sign file
-  stdin,stdout,stderr=ssh_client.exec_command(f"gpg --batch --passphrase {passphrase} --armor --detach-sig --default-key infra@sonarsource.com {directory}/{filename}")
+  exec_ssh_command(ssh_client, f"gpg --batch --passphrase {passphrase} --armor --detach-sig --default-key infra@sonarsource.com {directory}/{filename}")
   print(f'signed {directory}/{filename}')
-  stdin,stdout,stderr=ssh_client.exec_command(f"ls -al {directory}")
-  print(stdout.readlines())
+  exec_ssh_command(ssh_client, f"ls -al {directory}")
   ssh_client.close()
   return release_url
 
+def exec_ssh_command(ssh_client, command):
+  stdin,stdout,stderr=ssh_client.exec_command(command)
+  stdout_contents = '\n'.join(stdout.readlines())
+  print(f"stdout: {stdout_contents}")
+  stderr_contents = '\n'.join(stderr.readlines())
+  print(f"stderr: {stderr_contents}")
+  if stderr_contents:
```

Comment:
In our case, the ssh command was returning successfully, with details of the error in stderr